### ISSUE TITLE
Control not working when app submitted to store

### ISSUE THREAD

**Issue by RagBuster** [microsoft/rate-my-app#24]
I have implemented the control in to my app (MyDenon Receiver) and testing on the emulator and when deployed to my phone worked as expected but as soon as the app has been published to the store the control no longer works correctly.
The control reports that the launch count is 0 no matter how many times the app is started.
What I find strange is how it work perfectly during testing.

**Comment by ltomuta** [comment]
That is strange indeed. To be tested ...

**Comment by ltomuta** [comment]
I just tested one of the apps I knew to be using RMA and the control works just fine in the published version. The version of RMA used is probably 1.0.2. Are you using that version or the newer one, 1.1.0?

**Comment by RagBuster** [comment]
I am using 1.1.0 from Nuget

**Comment by ltomuta** [comment]
I tried the "Compass" app which I am told uses RMA 1.1.0 as well and  the component works just fine. I wonder if this is still a bug affecting only your application for some reason.

Can you send me the XAP file you have submitted to the Store?

**Comment by RagBuster** [comment]
I can certainly do this. How do I send it to you?

**Comment by ltomuta** [comment]
first dot last @ gmail dot com

**Comment by RagBuster** [comment]
I have emailed you the app.

**Comment by ltomuta** [comment]
Got it. I will try to have a look at it asap, but I'm currently really busy getting ready for a business trip.

**Comment by crea7or** [comment]
Same issue.

**Comment by crea7or** [comment]
Actually, one client seen the ratemyapp in the one problematic app and used the feedback. Don't know what to think. I was tested the app at three different devices and ratemyapp didn't fire.

**Comment by RagBuster** [comment]
I wonder the same. I have certainly noticed an increase to reviews especially 5 star but home come on 3 phones I tested on it did not work?

**Comment by RagBuster** [comment]
I have heard from a user that it is working in their region so wonder if this region related?

**Comment by ltomuta** [comment]
Sorry, I still have not had time yet to look at your package. What language & regional settings are you using and do you know what your user's settings might be?

**Comment by RagBuster** [comment]
My regional settings are UK but the working region was Germany. Still strange why it worked when deployed to my phone though.

**Comment by ltomuta** [comment]
Hmm, there may be something there.
I set my phone to use UK regional format and then I tried your app sideloaded (it worked) and installed from the store, trial mode, but this one did not work. I then bought the app and installed it and this time it worked just fine.

Could it be that in your tests & bug reports the failure is also only in the trial mode of your application, regardless of the regional settings? If so the behaviour is an undocumented feature as can be seen from https://github.com/nokia-developer/rate-my-app/blob/master/RateMyApp/Helpers/FeedbackHelper.cs#L134-L136

Basically it is by design that the user is expected to review only the fully functional version of the app and not the trial version.

**Comment by RagBuster** [comment]
This makes sense now. I was not aware that the launch count does not increment in the trial mode. Sorry you had to purchase the app to test.

**Comment by ltomuta** [comment]
No worries, it is good that we finally understand the root cause.
@crea7or, can you confirm if your problematic app is also a trial?

**Comment by crea7or** [comment]
Yes, this app and the second with problem are trials too.

**Comment by ltomuta** [comment]
Ok, so let's consider this issue closed.
